pokemongo_bot/step_walker.py

```diff
@@ -19,18 +20,21 @@ def __init__(self, bot, speed, dest_lat, dest_lng):
             dest_lng
         )
 
-        self.speed = speed
+        if self.bot.config.walk == 0:
+            self.speed = self.bot.config.walk_min - random() * (self.bot.config.walk_max - self.bot.config.walk_min)
```

Comment:
I think this should be
`self.speed = self.bot.config.walk_max - random() * (self.bot.config.walk_max - self.bot.config.walk_min)`
?